datadog/dogshell/wrap.py

```diff
@@ -86,17 +148,44 @@ def execute(cmd, cmd_timeout, sigterm_timeout, sigkill_timeout,
+quotes to prevent python as soon as there is a space in your command. \n \n\
+NOTICE: In normal mode, the whole stderr is printed before stdout, in flush_live mode they will \
+be mixed but there is not guarantee that messages sent by the command on both stderr and stdout \
+are printed in the order they were sent.")
+
+    parser.add_option('-n', '--name', action='store', type='string', help="the name of the event \
+as it should appear on your Datadog stream")
+    parser.add_option('-k', '--api_key', action='store', type='string',
+                      help="your DataDog API Key")
     parser.add_option('-m', '--submit_mode', action='store', type='choice',
-                      default='errors', choices=['errors', 'all'])
-    parser.add_option('-t', '--timeout', action='store', type='int', default=60 * 60 * 24)
-    parser.add_option('--sigterm_timeout', action='store', type='int', default=60 * 2)
-    parser.add_option('--sigkill_timeout', action='store', type='int', default=60)
-    parser.add_option('--proc_poll_interval', action='store', type='float', default=0.5)
-    parser.add_option('--notify_success', action='store', type='string', default='')
-    parser.add_option('--notify_error', action='store', type='string', default='')
+                      default='errors', choices=['errors', 'all'], help="[ all | errors ] If set \
+to error, an event will be sent only of the command exits with a non zero exit status or if it \
+times out.")
+    parser.add_option('-t', '--timeout', action='store', type='int', default=60 * 60 * 24,
+                      help="(in seconds)  a timeout after which your command must be aborted. An \
+event will be sent to your DataDog stream (default: 24hours)")
+    parser.add_option('--sigterm_timeout', action='store', type='int', default=60 * 2,
+                      help="(in seconds)  When your command times out, the \
+process it triggers is sent a SIGTERM. If this sigterm_timeout is reached, it will be sent a \
+SIGKILL signal. (default: 2m)")
+    parser.add_option('--sigkill_timeout', action='store', type='int', default=60,
+                      help="(in seconds) how long to wait at most after SIGKILL \
+                              has been sent (default: 60s)")
+    parser.add_option('--proc_poll_interval', action='store', type='float', default=0.5,
+                      help="(in seconds). interval at which your command will be polled \
+(default: 500ms)")
+    parser.add_option('--notify_success', action='store', type='string', default='',
+                      help="a message string and @people directives to send notifications in \
+case of success.")
+    parser.add_option('--notify_error', action='store', type='string', default='',
+                      help="a message string and @people directives to send notifications in \
+case of error.")
+    parser.add_option('-f', '--flush_live', action='store_true', dest='flush_live', default=False,
```

Comment:
Yeah, I was wondering as well but so far the behaviour was to have it set to false and I thought maybe some of our clients use dogwrap in some scripts of their owns. So basically I did that for "backward compatibility" but since dogwrap has been released very recently maybe we can put it to True by default. Shall I do that ?